### DISCUSSION TITLE
Before posting an issue : TypeError: can only concatenate str (not "list") to str

### DISCUSSION THREAD

**Discussion by ronbere** [vladmandic/sdnext#1484] [Q&A]
before posting an issue, I would like to know if anyone has had this error before, I did not make any changes, no new extensions installed, it happened suddenly...

CUDA SETUP: Loading binary C:\automatic\venv\lib\site-packages\bitsandbytes\libbitsandbytes_cudaall.dll...
╭───────────────────────────────────────── Traceback (most recent call last) ─────────────────────────────────────────╮
│ C:\automatic\launch.py:189 in <module>                                                                              │
│                                                                                                                     │
│   188                                                                                                               │
│ ❱ 189     instance = start_server(immediate=True, server=None)                                                      │
│   190     while True:                                                                                               │
│                                                                                                                     │
│ C:\automatic\launch.py:144 in start_server                                                                          │
│                                                                                                                     │
│   143         else:                                                                                                 │
│ ❱ 144             server = server.webui(restart=not immediate)                                                      │
│   145     if args.profile:                                                                                          │
│                                                                                                                     │
│ C:\automatic\webui.py:302 in webui                                                                                  │
│                                                                                                                     │
│   301     start_common()                                                                                            │
│ ❱ 302     start_ui()                                                                                                │
│   303     modules.sd_models.write_metadata()                                                                        │
│                                                                                                                     │
│ C:\automatic\webui.py:229 in start_ui                                                                               │
│                                                                                                                     │
│   228     startup_timer.record("before-ui")                                                                         │
│ ❱ 229     shared.demo = modules.ui.create_ui()                                                                      │
│   230     startup_timer.record("ui")                                                                                │
│                                                                                                                     │
│ C:\automatic\modules\ui.py:1446 in create_ui                                                                        │
│                                                                                                                     │
│   1445     interfaces += [(settings_interface, "Settings", "settings")]                                             │
│ ❱ 1446     extensions_interface = ui_extensions.create_ui()                                                         │
│   1447     interfaces += [(extensions_interface, "Extensions", "extensions")]                                       │
│                                                                                                                     │
│ C:\automatic\modules\ui_extensions.py:396 in create_ui                                                              │
│                                                                                                                     │
│   395                 update_extension_list()                                                                       │
│ ❱ 396                 extensions_table = gr.HTML(refresh_extensions_list_from_data(search_text                      │
│   397                 check.click(                                                                                  │
│                                                                                                                     │
│ C:\automatic\modules\ui_extensions.py:335 in refresh_extensions_list_from_data                                      │
│                                                                                                                     │
│   334         tags_string = ' '.join(tags)                                                                          │
│ ❱ 335         tags = tags + ["installed"] if installed else tags                                                    │
│   336         if len([x for x in tags if x in hide_tags]) > 0:                                                      │
╰─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
TypeError: can only concatenate str (not "list") to str
Appuyez sur une touche pour continuer...

launching with --safe doing the same issue

**Comment by vladmandic** [comment]
haven't seen it but could be due to recent change i made to format of extension index, although i hope not.
i'll check, but it will have to wait until tomorrow.
if you don't hear otherwise and issue still persists, please create an issue.

**Reply by ronbere** [reply]
of course I can wait, no problem, thank you for all the work provided. it doesn't seem to want to start the server. Take your time

**Reply by vladmandic** [reply]
Try to delete html/extensions.json so it does forced refresh?

**Reply by Hexag0nSun** [reply]
Thanks, had the same problem after upgrade. This solved it.

**Comment by vis223** [comment]
I'm struggling with this too.  The last thing I did was 'refresh extensions' then 'update' and have been locked out by the concatenate message ever since

**Comment by Davikar** [comment]
Same here... it was working fine, then I updated the extensions and restarted.

**Comment by vis223** [comment]
> Try to delete html/extensions.json so it does forced refresh?

This worked for me. Thank you.

**Reply by ronbere** [reply]
you can update extensions??

**Reply by vis223** [reply]
no, but no longer locked out. no doubt a fix will come soon

**Reply by vladmandic** [reply]
Check the thread - I've posted a fix 1h ago.

**Comment by vladmandic** [comment]
fixed.

**Comment by cleverestx** [comment]
This just happened to me now while trying to install ONE BUTTON PROMPT addon, the install gave me this error and I had to delete that extension JSON file to get back in...I guess that extension isn't supported...neither is the extension ROOP...what a bummer!
